docs/rules/no-unknown-property.md

```diff
@@ -22,6 +22,16 @@ var React = require('react');
 var Hello = <div className="hello">Hello World</div>;
 ```
 
+## Rule Options
+
+HTML properties and attributes which would normally be considered warnings can be whitelisted. If you wanted to allow use of 'class' and 'for' attributes because your Babel configuration handles transforming them to DOM attributes, the configuration would be:
+
+```js
+"rules": {
+  "react/no-unknown-property": [1, { "allow": ["class", "for"] }],
```

Comment:
As far as I know, React itself doesn't support "class" or "for", and requires "className" and "htmlFor". Can you point to some documentation that allows for this usage?